sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -275,10 +285,19 @@ export class EventHubClient {
   }
 
   /**
-   * Creates an EventHubProducer that can be used to send events to the Event Hub for which this client
-   * was created.
+   * Creates an Event Hub producer responsible for transmitting `EventData` to the Event Hub.
```

Comment:
`transmitting` feels heavy

Reply:
I like `sending` since we have a `send` method.